Firestore/core/src/local/leveldb_transaction.h

```diff
@@ -155,6 +155,17 @@ class LevelDbTransaction {
    */
   void Delete(absl::string_view key);
 
+  /**
+   * Remove the database entry (if any) for all "key" starting with given
+   * prefix.  It is not an error if "key" did not exist in the database. When
+   * function returns true, means the function has finished execution. If it
```

Comment:
The function returns true when all keys are deleted (or no key to delete), otherwise it returns false.